config/config.yaml

```diff
@@ -498,7 +498,7 @@ solving:
             2030: 99   # Wind-an-Land Law 2028
             2035: 115   # Wind-an-Land Law 2030
```

Comment:
We should specify a smoother trajectory for onwind buildout, to avoid the potential of a jump in from 115 to 160 in 45. This will require double-checking all corresponding min, max constraints in config.yaml and scenarios.yaml to avoid accidental infeasibilities